README.md

```diff
@@ -27,8 +27,16 @@ After replacing these values, run:
 ```sh
 NETWORK='rinkeby'
 GAS_PRICE_WEI='1000000000'
+INFURA_KEY="Your infura key"
 yarn deploy --network $NETWORK --gasprice $GAS_PRICE_WEI
 ```
 
 New files containing details of this deployment will be created in the `deployment` folder.
 These files should be committed to this repository.
+
+In order to verify the contract in Etherscan, run:
+
+```
+ETHERSCAN_API_KEY="your key"
+npx hardhat verify --network $NETWORK DEPLOYED_CONTRACT_ADDRESS "Constructor argument 1"
```

Comment:
Very minor: when verifying the first time I replanced "Constructor argument 1" with the constructor arguments and left `DEPLOYED_CONTRACT_ADDRESS` where it was because it looked like it would be replaced automatically like for `$NETWORK`. My proposal makes the style of all parameters that the user has to change consistent.

```suggestion
npx hardhat verify --network $NETWORK "Address of deployed contract" "Constructor argument 1"
```